ModeratorFrontEnd/AIForOrcas/AIForOrcas.Client.BL/Services/DetectionService.cs

```diff
@@ -76,6 +76,8 @@ public async Task UpdateRequestAsync(DetectionUpdate request)
 			var dataJson = JsonSerializer.Serialize(request);
 			var stringContent = new StringContent(dataJson, Encoding.UTF8, "application/json");
 
+			// TODO: Catch authentication issues
```

Comment:
Please address TODO

Reply:
Michelle. You are correct that this is a large PR. The one I was originally working on was even larger, but I bit off more than I could chew and had to scale back. I was attempting to get us to a stable baseline before everyone started hacking on Monday. A full 70% was me making "cosmetic" changes to bring the solution up to .NET 6 (namespaces, global usings, collapsing Programs.cs/Setup.cs, etc.) and updating the various libraries. The rest was primarily focused on updating the security mechanisms where if I updated the API, I had to update the UI as well. Probably could have split into a PR for the .NET stuff and a PR for the security stuff, but that would have meant waiting for the first PR to be approved before I could move on to the 2nd. I do appreciate the reminder about the TODO: and will address first thing tomorrow.